src/core/result_cache.jl

```diff
@@ -47,11 +47,11 @@ end
 """
 Return true if the data for `timestamp` is stored in cache.
 """
-function is_cached!(cache::ResultCache, type, name, stage, timestamp)
-    return is_cached!(cache, make_cache_key(stage, type, name), timestamp)
+function is_cached(cache::ResultCache, type, name, stage, timestamp)
```

Comment:
A docstring was referencing `is_cached`, which also caused building of docs to fail. The name should not have a !